ISICkonto/Flow/ViewControllers/BalanceViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  BalanceViewController.swift
+//  ISICkonto
+//
+//  Created by Vendula Švastalová on 14/02/2019.
+//  Copyright © 2019 Vendula Švastalová. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SVProgressHUD
+
+class BalanceViewController : UIViewController {
+    
+    private var balanceLabel: UILabel!
+    private var textLabel: UILabel!
+    private var refreshButton: UIButton!
+    private var logOutButton: UIButton!
+    private var buttonView: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        createConstraints()
+    }
+  
+    func setupUI() {
+        self.view.backgroundColor = UIColor(red: 93/255, green: 149/255, blue: 170/255, alpha: 1)
+        
+        balanceLabel = UILabel()
+        balanceLabel.textAlignment = .center
+        balanceLabel.font = UIFont(name: "Cabin-Bold", size: 85)!
+        balanceLabel.textColor = UIColor.white
+        
+        textLabel = UILabel()
+        textLabel.textAlignment = .center
+        textLabel.font = UIFont(name: "Cabin-Regular", size: 25)!
+        textLabel.textColor = UIColor(red: 182/255, green: 220/255, blue: 252/255, alpha: 1)
+        textLabel.text = "Your balance is".localized
+        
+        refreshButton = UIButton()
+        refreshButton.setImage(UIImage(named: "reloadIcon"), for: .normal)
+        
+        logOutButton = UIButton()
+        logOutButton.setImage(UIImage(named: "accountIcon"), for: .normal)
+        
+        buttonView = UIView()
+        
+        view.addSubview(balanceLabel)
+        view.addSubview(textLabel)
+        buttonView.addSubview(refreshButton)
+        buttonView.addSubview(logOutButton)
+        view.addSubview(buttonView)
+        
+    }
+    
+    func createConstraints() {
+        textLabel.snp.makeConstraints { make in
+            make.bottom.equalTo(balanceLabel.snp.top).offset(-10)
+            make.centerX.equalTo(balanceLabel.snp.centerX)
```

Comment:
když přidáváš constraint na stejné atributy, není potřeba ho potom specifikovat v tom argumentu. Stejný výsledek bude mít zápis `make.centerX.equalTo(balanceLabel)`. Kratší, hezčí a prsty to tolik nebolí 😊

Reply:
to je super, to jsem nevedela 👍